Makefile

```diff
@@ -2,6 +2,9 @@ PREFIX := /usr/local
 VERSION := $(shell git describe --exact-match --tags 2>/dev/null)
 BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
 COMMIT := $(shell git rev-parse --short HEAD)
+GOFILES ?= $(shell find . -name '*.go')
```

Comment:
This does not work on Windows systems with MinGW. You have to use double quotes to get it working:
```
GOFILES ?= $(shell find . -name "*.go")
```